installer/manifests/keptn/charts/control-plane/templates/core.yaml

```diff
@@ -570,6 +570,90 @@ spec:
     app.kubernetes.io/instance: {{ .Release.Name }}
 ---
 # configuration-service
+{{- if .Values.newConfigurationService.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: configuration-service
+  labels:
+    app.kubernetes.io/name: configuration-service
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/part-of: keptn-{{ .Release.Namespace }}
+    app.kubernetes.io/component: {{ include "control-plane.name" . }}
+    app.kubernetes.io/version: {{ .Values.newConfigurationService.image.tag | default .Chart.AppVersion }}
+    helm.sh/chart: {{ include "control-plane.chart" . }}
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: configuration-service
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  replicas: 2
```

Comment:
let's make the number of replicas configurable via the helm chart's `values.yaml`